lib/gwdb-legacy/dutil.mli

```diff
@@ -119,3 +125,6 @@ val map_family_ps :
     - Generic type used to represent another large part of information of [Def.gen_family]
     into another one.
     If [fd] is present, apply it on it on every date (marriage, divorce, famillial events, etc.).*)
+
+val insert_lowered_string_istrs :
```

Comment:
Could you please find a better name for this function? `insert_lowered_name_suffix_istrs`?